book/modules.md

```diff
@@ -197,8 +199,7 @@ Submodules are modules inside modules. They are automatically created when you c
 
 The difference is that `export module some-module` _only_ adds the module as a submodule, while `export use some-module` _also_ re-exports the submodule's definitions. Since definitions of submodules are available when importing from a module, `export use some-module` is typically redundant, unless you want to re-export its definitions without the namespace prefix.
 
-> **Note**
-> `module` without `export` defines only a local module, it does not export a submodule.
+> **Note** > `module` without `export` defines only a local module, it does not export a submodule.
```

Comment:
Recommend converting this to a `::: tip` - See https://v1.vuepress.vuejs.org/guide/markdown.html#custom-containers

If not, at least recommend changing the second ">" to a colon.

Reply:
I didn't even edit this myself, it must have been the pre-commit formatter :sweat_smile: 

I agree with your assessment though, I just switched it to be a tip